andela/ah-orcas-frontend
# #161254610 setup redux folder/file structure

#### What does this PR do?
- setup redux folder/file structure redux

#### Description of Task to be completed?
- Create the appropriate folder/file structure for React/Redux workflow

#### How should this be manually tested?
- Clone this repository 
```git clone https://github.com/andela/ah-orcas-frontend.git```
- Change the working directory to the project folder
``` cd ah-orcas-frontend```
- Checkout the setup redux branch
``` git checkout ch-setup-redux-161254610```
- Install the required dependencies:
```npm install```
- Run the local development server:
```npm start```
- You should be able to view the ah-orcas-frontend-application on your browser

#### Any background context you want to provide?
- The current folder structure is the one that was bootstrapped by create-react-app. Creating the React/Redux folder structure makes the code more readable and easier to maintain.

#### What are the relevant pivotal tracker stories?
[#161254610](https://www.pivotaltracker.com/story/show/161254610)

#### Checklist:
